clash-lib/prims/common/Clash_Class_BitPack.primitives.yaml

```diff
@@ -1,15 +1,71 @@
 - BlackBox:
-    name: Clash.Class.BitPack.Internal.packFloat#
+    name: Clash.Class.BitPack.Internal.packInt8#
     kind: Expression
-    type: 'packFloat# :: Float
-      -> BitVector 32'
+    type: 'packInt8# :: Int8
+      -> Bitvector 8'
     template: ~ARG[0]
     workInfo: Never
 - BlackBox:
-    name: Clash.Class.BitPack.Internal.unpackFloat#
+    name: Clash.Class.BitPack.Internal.packInt16#
     kind: Expression
-    type: 'packFloat# :: BitVector
-      32 -> Float'
+    type: 'packInt16# :: Int16
+      -> Bitvector 16'
+    template: ~ARG[0]
```

Comment:
Hmm, I doubt this is true for VHDL. Aren't `Int`s typed as `signed`?

Reply:
I was also under the impression int is signed and word is unsigned in netlist / HDL